### ISSUE TITLE
Extreme performance issues with Repeating effect

### ISSUE THREAD

**Issue by Mirayii** [GC-spigot/AdvancedEnchantments#3533]
### Describe the bug

Even if we're not using any Repeating effect the performance of the server is rapidly going down when Attacking or doing anything.
We did a test and removed all other plugins that could cause the problem (reading from spark), yet it did not help and the TPS went up only after disabling AdvancedEnchantments. It would be nice to have ability to completely disable Repeating effects in config. It's not needed in most cases and the performance impact is extreme with higher count players.

### How to reproduce

Let a lot of players (30+) play on the server, do things, attack entities or just walk and the TPS will drop to 12 and lower in a short while.

### Screenshots / Videos

![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/45139953/08b5aeef-f369-4e68-9758-4530a641bc73)

### "/ae plinfo" link

https://paste.md-5.net/terabufuro

**Comment by Mirayii** [comment]
https://spark.lucko.me/6PK1IDqV1Z
